doc/src/configuration.md

```diff
@@ -111,6 +111,16 @@ or at `~/.config/stylix/palette.html` on Home Manager.
 
 ## Fonts
 
+Fonts can apply in three ways:
+
+1. An app's own font options
+  (e.g. [the Alacritty module](/options/modules/alacritty.html) sets home-manager's
+  `programs.alacritty.settings.font`).
+1. Widely, through
+  [the Fontconfig module](/options/modules/fontconfig.html).
+1. Made available in an environment using
+  [the font packages module](/options/modules/font-packages.html).
+
```

Comment:
@Flameopathic my goal with expanding the general fonts documentation was to provide an overview. Perhaps this implementation achieves that without being extraneous?

Also to be able to link to this overview from the documentation of fontconfig and font-packages.